src/index.ts

```diff
@@ -149,9 +151,18 @@ export async function main(
       coverageFilePaths = args.file
     }
   }
+  const isNegated = (path: string) => path.startsWith('!')
```

Comment:
Please write a test to confirm this behaves as expected.

Reply:
Since `isNegated` is just a helper function for the anonymous arrow function a bit below, its behavior should be caught by the E2E test I pushed in one of my more recent commits.